lib/mumukit/templates/file_hook.rb

```diff
@@ -65,8 +65,9 @@ def self.isolated(value=true)
       end
     end
 
-    def self.mashup(*args)
+    def self.mashup(*args, &map_block)
       include Mumukit::Templates::WithMashupFileContent
+      define_method(:map_fields, &map_block) if block_given?
```

Comment:
I think this method name is too generic. I'd rename this method as `map_mashup_fields` in order to make it harder to override it by accident.

Reply:
Roger!